### DISCUSSION TITLE
Deployer not removing folders/files

### DISCUSSION THREAD

**Discussion by Chrisedmo** [deployphp/deployer#2300] [Help needed]
- Deployer version:  6.8.0
- Deployment OS: 10.14

I am using deployer and I have recently deleted a folder full of media files, and when I deploy they're still on the server? They not in my git the media folder is empty. Why is it not removing the folder form my server?

I have set the removed folder as a shared item - so it's outside the directory.

**Comment by antonmedv** [comment]
Restart server.
